demo/admin/src/products/ProductForm.tsx

```diff
@@ -84,6 +94,12 @@ function ProductForm({ id }: FormProps): React.ReactElement {
               ...filter<GQLProductFormManualFragment>(productFormFragment, data.product),
               price: String(data.product.price),
               image: rootBlocks.image.input2State(data.product.image),
+              packageDimensions: {
+                  ...data.product.packageDimensions,
```

Comment:
This spread is unnecessary as there's nothing in the dimensions except those three fields. But I believe you want to highlight the required transformations (e.g., a string would be perfectly fine in the spread).